core/src/main/scala/org/http4s/parser/AdditionalRules.scala

```diff
@@ -158,11 +158,12 @@ private[http4s] trait AdditionalRules extends Rfc2616BasicRules { this: Parser =
       min: Int,
       sec: Int,
       wkday: Int): HttpDate =
-    Try(
+    Try {
       org.http4s.HttpDate.unsafeFromZonedDateTime(
-        ZonedDateTime.of(year, month, day, hour, min, sec, 0, ZoneOffset.UTC))).getOrElse {
+        ZonedDateTime.of(year, month, day, hour, min, sec, 0, ZoneOffset.UTC))
+    }.getOrElse {
       // TODO Would be better if this message had the real input.
-      throw new ParseFailure("Invalid date", s"$year-$month-$day $hour:$min:$sec")
+      throw new ParseFailure("Invalid date", s"$wkday $year-$month-$day $hour:$min:$sec")
```

Comment:
This `wkday` is irrelevant, but I couldn't figure out how to get the parsers to ignore it.  I want a non-capturing version of the `Wkday` rule, so we can drop that parameter.

Reply:
lol. I'm not sure that's the best way to handle it, but it does work.

Reply:
We should probably validate that the wkday was correct for the rest of the date.